Source/Core/Common/IOFile.cpp

```diff
@@ -101,6 +101,15 @@ bool IOFile::Close()
   return m_good;
 }
 
+std::unique_ptr<IOFile> IOFile::Duplicate(const char openmode[]) const
```

Comment:
Can't this return `IOFile` instead of `std::unique_ptr<IOFile>`?

Reply:
Can it? It would certainly make things easier but I thought I needed to maintain this pointer.

Reply:
You created the pointer just now. I don't see why you would need to maintain it.